packages/react-scripts/package.json

```diff
@@ -1,18 +1,18 @@
 {
-  "name": "react-scripts",
-  "version": "3.0.1",
-  "description": "Configuration and scripts for Create React App.",
+  "name": "browser-extension-react-scripts",
```

Comment:
This is important to be able `npm publish` as per [this guide](https://codeburst.io/customizing-create-react-app-done-right-4a22683f2e09).